### PR TITLE
Reword a complex sentence on EF page

### PR DESCRIPTION
**Pull request by bskrksyp9**
Reframed a sentence

## Description
Sentence correction done on line number 14 to match the context, perhaps this could be even more re-framed

### CODE REVIEW

File: src/content/foundation/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ lang: en
 
 The [Ethereum Foundation](http://ethereum.foundation/) (EF) is a non-profit organization dedicated to supporting [Ethereum](/what-is-ethereum/) and related technologies.
 
-The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, nor are they the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).
+The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, nor they are the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).
```
Comment (minimalsm) [inline src/content/foundation/index.md:14]:
```suggestion
The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, and they aren't the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @bskrksyp9 thanks for the PR. The changes you've made changes the meaning of the sentence from saying *the EF **aren't** the only organization that funds critical to saying the opposite.

But I think you've highlighted something. The use of formal conjunction here (*'nor'*) could be confusing to non-native English speakers. I've suggested a slight improvement.

File: src/content/foundation/index.md (hunk 1 above)
Comment (bskrksyp9) [reply src/content/foundation/index.md:14]: yes thanks !

**Review by bskrksyp9 — COMMENTED** [review]
Thanks for improving it !

**Pull request merged**